home-exercises/src/main/java/advancedtest/exercise2/Basket.java

```diff
@@ -0,0 +1,42 @@
+package advancedtest.exercise2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket<T> {
+    private T items;
```

Comment:
Kāda ir šī mainīgā nozīme?

Reply:
Pēdējā nodarbībā runājām, šis bija galvā palicis no generics piemēriem, taču nevajadzēja galu galā izmantot un arī izdzēst piemirsās